frontend/src/components/homeview/homeview_components/DashboardStudyKing.jsx

```diff
@@ -1,8 +1,11 @@
 import React, { Component } from "react";
 import LineChartTimeStream from "./linechart_time_stream.js";
+import StudyKingService from "../../services/StudyKingService";
 class DashboardStudyKing extends Component {
   state = {
-    data: [
+    winner: "some user",
+    winnerMinutes: "some time",
```

Comment:
- [ ] IMO it makes more sense to save the winnerMinutes as type double instead of string, I believe this data will later surely be used somewhere else in arithmetical operations.

Reply:
In fact, it is not double but str. eg. "1hour 20min"  "2 hours 25 min". But I agree that all these kind of duration data should be set to double as "Seconds". Nice Issue for kick-off though